add_gif.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+USAGE="Usage: ./add_gif.sh name http://url_of_the_gif"
+NAME=$1
+URL=$2
+
+# Usage 
+if [[ "$NAME" == "--help" ]]; then
+    echo $USAGE
+    exit 0
+fi
+
+if [[ $# -ne 2 ]]; then
+    echo $USAGE
+    exit 1
+fi
+
+# Ensure availability of wget
+if ! type wget > /dev/null 2> /dev/null; then
+    echo "Script requires wget"
+    exit 2
+fi
+
+# Ensure name is not already in use
+if [[ -f ./_orig/$NAME.gif ]]; then
```

Comment:
This is going to explode if $NAME has a space in it, but in that case I don't know if the output would work anyway? Maybe we should prevent names with spaces and other strange characters?